.gitignore

```
@@ -1,3 +1,13 @@
 tool_test_output.html
 tool_test_output.json
 .DS_Store
+.DS_Store
+workflows/.DS_Store
+workflows/proteomics/.DS_Store
+workflows/proteomics/clinicalmp/.DS_Store
+workflows/proteomics/encyclopedia/.DS_Store
+workflows/proteomics/encyclopedia/.DS_Store
+workflows/proteomics/encyclopedia/.DS_Store
+workflows/proteomics/encyclopedia/.DS_Store
+workflows/proteomics/encyclopedia/.DS_Store
+workflows/proteomics/encyclopedia/.DS_Store
```

This doesn't do anything, the first entry recursively prevents adding .DS_Store files. It doesn't help with those files that were already added.
```suggestion
.DS_Store
```